web/clusters.go

```diff
@@ -573,26 +571,6 @@ func NewClusterHandler(client consul.Client, s services.ChecksService) gin.Handl
 			return
 		}
 
-		selectedChecks, getCheckErr := s.GetSelectedChecksById(clusterId)
-		if getCheckErr != nil {
-			StoreAlert(c, NoCheckSelected())
-		}
-
-		connectionData, getConnErr := s.GetConnectionSettingsById(clusterId)
-		defaultConnectionData, getDefConnErr := getDefaultConnectionSettings(client, clusterItem)
```

Comment:
We would need to remove the `getDefaultConnectionSettings` and `getChecksCatalogWithSelected` functions implementations, as they are done in the frontend. 
By the way, you could get the default settings using this consul code. As you already used consul data to create the connections settings.
Whether we put all the data with consul or we wait until having the new projections, but let's do not do half of the work hehe

PD: Most probably, the `Alert` functions removed here are not used elsewhere, so they could be removed too

Reply:
I will hold the works on this because we're in the process to get the hosts projected, you're right, we shouldn't rush :muscle: